setup.cfg

```diff
@@ -1,25 +1,7 @@
-[flake8]
-max-line-length = 90
-ignore = F403,E226,E731,E275,W503,F405,E722,E741,W504,W605,E402
-exclude = .github
-
-[flake8_nb]
-max-line-length = 90
-ignore = F403,E226,E731,E275,W503,F405,E722,E741,W504,W605,E402
-exclude = .github
-
-[tool:pytest]
-python_files = tests/*test_*.py docs/*test_*.py
-python_classes = Test_*
-python_functions = test_*
-addopts = --durations=20 --maxfail=5
 
 [versioneer]
 VCS = git
 style = pep440
 versionfile_source = xdsl/_version.py
 versionfile_build = xdsl/_version.py
 tag_prefix = v
-
-[aliases]
-test = pytest -v
```

Comment:
Where has this gone?

Reply:
Aliases in setup.cfg are actually aliases to pytest, not geral commands. So this alias is invoked with `python setup.py test` and is equivalent to `python setup.py pytest -v`, which I don't think actually does anything. I'm not sure why it was added in the first place.

https://stackoverflow.com/questions/63672256/adding-aliases-fails-with-error-invalid-command-xyz